functions.sh

```diff
@@ -1013,7 +1013,7 @@ validate_vars() {
             graph_error "ERROR: CentOS 32bit doesn't support xfs on partition /"
             return 1
           fi
-          for j in `seq 1 $PART_COUNT`; do
+          for j in $(seq 1 $PART_COUNT); do
```

Comment:
This is bash not C. I do not think that c-style for-loops are more common.

Am Freitag, 12. Februar 2016 schrieb Silvio Knizek :

> In functions.sh
> <https://github.com/virtapi/installimage/pull/59#discussion_r52795132>:
>
> > @@ -1013,7 +1013,7 @@ validate_vars() {
> >              graph_error "ERROR: CentOS 32bit doesn't support xfs on partition /"
> >              return 1
> >            fi
> > -          for j in `seq 1 $PART_COUNT`; do
> > +          for j in $(seq 1 $PART_COUNT); do
>
> seq is an own program. Doing it in C-style is much more common.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/virtapi/installimage/pull/59/files#r52795132>.
>